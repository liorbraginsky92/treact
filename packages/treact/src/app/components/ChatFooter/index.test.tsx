import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { ChatFooter } from './'

it('Message renders correctly', () => {
  const noop = () => null
  const tree = renderer.create(
    <ChatFooter value="Hello, world!" change={noop} submit={noop} />,
  ).toJSON()

  expect(tree).toMatchSnapshot()
})

