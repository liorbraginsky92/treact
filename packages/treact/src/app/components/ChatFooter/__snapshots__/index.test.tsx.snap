// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders correctly 1`] = `
<div
  className="css-yneirn"
>
  <textarea
    className="css-1htpcuh"
    onChange={[Function]}
    placeholder="  Write a message"
    rows={1}
    style={undefined}
    value="Hello, world!"
  />
  <button
    className="css-grs26u"
    onClick={[Function]}
  >
    Send
  </button>
</div>
`;
